tests/ast/data/DataFrame.unpivot.test.DISABLED

```diff
@@ -0,0 +1,12 @@
+## TEST CASE
+
```

Comment:
maybe let's add ticket to fix this?

Reply:
Filed https://snowflakecomputing.atlassian.net/browse/SNOW-1663699